hw/ip/otp_ctrl/data/otp_ctrl.hjson

```diff
@@ -985,14 +985,53 @@
     { name: "PART.DATA_REG.INTEGRITY",
       desc: "All partition buffer registers are protected with ECC on 64bit blocks."
     }
+    { name: "PART.DATA_REG.BKGN_CHK",
+      desc: "The digest of buffered partitions is recomputed and checked at pseudorandom intervals in the background."
+    }
+    { name: "PART.MEM.REGREN"
+      desc: "Unbuffered ('software') partitions can be read-locked via a CSR until the next system reset."
+    }
+    { name: "PART.MEM.SW_UNREADABLE"
+      desc: "Secret buffered partitions become unreadable to software, once they are locked via the digest."
+    }
+    { name: "PART.MEM.SW_UNWRITABLE"
+      desc: "All partitions become unwritable by software, once they are locked via the digest."
+    }
+    { name: "LC_PART.MEM.SW_UNREADABLE"
+      desc: "The life cycle partition is not directly readable via software."
+    }
+    { name: "LC_PART.MEM.SW_UNWRITABLE"
+      desc: "The life cycle partition is not directly writable via software."
+    }
```

Comment:
I wonder whether we should have a `SW_NOACCESS` name or similar. It seems a bit silly to have to duplicate things like this.

Reply:
Well, the reason for splitting this out is that we do have instances of not-readable-but-writable by SW, for instance the AES key register...

Reply:
Ah, I see. But I still think it might be worth adding the "NOACCESS" version too, to avoid boilerplate in the common case.

Reply:
Ah, yes we can do that.